Automattic/wp-calypso
# Update/videopress onboarding v2 processing step

#### Proposed Changes

This PR converts the site creation step to a processing step.

#### Testing Instructions

* Apply patch
* Navigate to `http://calypso.localhost:3000/setup/intro?flow=videopress`
* Complete the flow until plan selection
* Select a plan
* ✅ You should be redirected to the processing step
* Wait until it's done
* ✅ You should be redirected to the checkout page
* ✅ The products you selected should appear in the cart

Fix Automattic/greenhouse#1347
